### ISSUE TITLE
[ Adicionar Post ] Alguns erros

### ISSUE THREAD

**Issue by guilessa** [Social-Pipe/social-docker#48]
1. Os posts são adicionados mesmo com o erro na imagem, daí fica ativo na lista só que sem a imagem de capa quando da erro;

![Captura de tela 2022-02-10 055630](https://user-images.githubusercontent.com/25805990/153372871-f2022417-8d00-4c16-9dae-131572d6e2b3.png)

2. Imagina aleatória dando erro. É uma arte comum, mas ela não funciona. Outras imagens com outros formatos funcionam mas essa não, ela está em anexo para teste;

![TesteArte](https://user-images.githubusercontent.com/25805990/153372951-df6b6af9-5757-426c-ac7d-790289493b4a.png)

3. Limite de upload por arte e vídeo MUITO pequeno. Existe alguma solução?

**Comment by filiperochalopes** [comment]
@guilessa Já conversamos bastante sobre a questão de limite de arte e vídeo. A solução plausível seria mudar de hospedagem para que você pudesse pagar de acordo com o nível de HD necessário. A outra opção viável seria a compactação de arquivos, mas que nessa dimensão, como creio ter sido comentado também, exigiria muito processador. Então estamos com um MVP, certamente quando você escalar precisará de uma hospedagem com maior HD e processador o que pode ser BEM caro. Os recursos são limitados, o que acho não ter problemas para um projeto alpha, porém se desejar migrar poderemos fazer isso para ti.

**Comment by guilessa** [comment]
Fala time, bom dia! 

Já temos a alteração concluída?

**Comment by guilessa** [comment]
@filiperochalopes essa é a melhor solução? Não conseguimos usar algo expansível de acordo com o uso apenas para os assets?

**Comment by filiperochalopes** [comment]
> @filiperochalopes essa é a melhor solução? Não conseguimos usar algo expansível de acordo com o uso apenas para os assets?

Sim, na hospedagem atual não temos como fazer nenhuma dessas alterações. Existem sim hospedagens que trabalham com hds que podem ser expandidos. Você pode optar por um desses.

**Comment by guilessa** [comment]
> Simplemente não tive resposta tua durante 13 dias @guilessa nem por email, nem whatsapp, nem por aqui. Nenhum dos 3 meios de contato estabelecidos há quase duas semanas. Por falta de retorno decidimos por aguardar a sua validação quando ao procedimento das correções.
> 
> Justamente por isso estava marcado como `question`, issues marcadas com essa tag exigem uma replica para compreensão do escopo da alteração. Até porque em uma issue apenas foram abertos 3 problemas. Faremos o seguinte, corrigiremos os 2 primeiros, confere?

Qual resposta você precisava? Pensei que ao colocar o erro aqui, era para entrar na fila de correção. Sempre foi assim, não entendi.

**Comment by filiperochalopes** [comment]
No momento mais nenhuma, as alterações já estão em andamento sendo desenvolvidas por @lucasnetwork . No momento anterior precisava de uma resposta de que tinha compreendido o que seria feito.

**Comment by guilessa** [comment]
Boa tarde! O erro permanece, também com essa arte aqui.

![Arte__Dia--07](https://user-images.githubusercontent.com/25805990/156639204-5be24b77-9809-43d2-ba5c-d4aa01e83127.png)
